### PR TITLE
Add SqueezeBERT

### PR DESCRIPTION
**Pull request by renmada**
See README.md

### CODE REVIEW

File: examples/language_model/squeezebert/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| :----:| :----:| :----:| :----:|
+|paddle|137s|186s|1.36|
+|pytorch|112s|172s|1.54|
+
+### 使用dataloader的情况
+#### paddle跑squeezebert
+```
+export CUDA_VISIBLE_DEVICES=0
+python -u ./run_glue.py \
+--model_type bert \
```
Comment (smallv0221) [inline examples/language_model/squeezebert/README.md]: --model_type squeezebert \

**Review by smallv0221 — COMMENTED** [review]
Please change your codes according to the comments in this pr https://github.com/PaddlePaddle/PaddleNLP/pull/872
And don't push those Irrelevant files.

**Comment by renmada** [comment]
> Irrelevant files

What is Irrelevant files？

**Comment by smallv0221** [comment]
> > Irrelevant files
> 
> What is Irrelevant files？

image
These empty files.

**Comment by renmada** [comment]
> Please change your codes according to the comments in this pr #872
> And don't push those Irrelevant files.

I have changed the code. But I have no idea about where the Irrelevant files came from

File: paddlenlp/transformers/squeezebert/modeling.py [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+
+        output_dict = {"feature_map": layer_output}
+        if output_attentions:
+            output_dict["attention_score"] = att["attention_score"]
+
+        return output_dict
+
+
+class SqueezeBertEncoder(nn.Layer):
+    def __init__(self, embedding_size, hidden_size, intermediate_size, num_attention_heads,
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/modeling.py:247]: The indent here is not correct

File: paddlenlp/transformers/squeezebert/tokenizer.py [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+            Default: True.
+        unk_token (str): The special token for unkown words. Default: "[UNK]".
+        sep_token (str): The special token for separator token . Default: "[SEP]".
+        pad_token (str): The special token for padding. Default: "[PAD]".
+        cls_token (str): The special token for cls. Default: "[CLS]".
+        mask_token (str): The special token for mask. Default: "[MASK]".
+    Examples:
+        .. code-block:: python
+            from paddlenlp.transformers import SqueezeBertTokenizer
+            tokenizer = SqueezeBertTokenizer.from_pretrained('SqueezeBert-small-discriminator')
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/tokenizer.py]: Unknown name `SqueezeBert-small-discriminator`

File: paddlenlp/transformers/squeezebert/modeling.py [hunk 4]
```diff
@@ -0,0 +1,691 @@
...
+            output_attentions=output_attentions,
+            output_hidden_states=output_hidden_states,
+        )
+        sequence_output = encoder_outputs[0]
+        pooled_output = self.pooler(sequence_output)
+
+        return (sequence_output, pooled_output) + encoder_outputs[1:]
+
+
+# class SqueezeBertForPretraining(SqueezeBertPreTrainedModel):
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/modeling.py]: Delete this if not needed

File: paddlenlp/transformers/squeezebert/modeling.py [hunk 5]
```diff
@@ -0,0 +1,691 @@
...
+                        self.squeezebert.config["initializer_range"],
+                        shape=layer.weight.shape))
+        elif isinstance(layer, nn.LayerNorm):
+            layer._epsilon = 1e-12
+
+
+@register_base_model
+class SqueezeBertModel(SqueezeBertPreTrainedModel):
+    """
+        Args:
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/modeling.py]: Incorrect indent.

File: paddlenlp/transformers/squeezebert/modeling.py [hunk 6]
```diff
@@ -0,0 +1,691 @@
...
+    def forward(
+            self,
+            input_ids=None,
+            attention_mask=None,
+            token_type_ids=None,
+            position_ids=None,
+            output_attentions=None,
+            output_hidden_states=None,
+    ):
+
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/modeling.py]: Need more doc here.

File: paddlenlp/transformers/squeezebert/modeling.py [hunk 7]
```diff
@@ -0,0 +1,691 @@
...
+            position_ids=None,
+            attention_mask=None)
+        logits = self.classifier(sequence_output)
+        logits = paddle.transpose(logits, perm=[2, 0, 1])
+        start_logits, end_logits = paddle.unstack(x=logits, axis=0)
+        return start_logits, end_logits
+
+
+class SqueezeBertForTokenClassification(SqueezeBertPreTrainedModel):
+
```
Comment (smallv0221) [inline paddlenlp/transformers/squeezebert/modeling.py:753]: Need more doc for models for specific tasks

**Review by smallv0221 — CHANGES_REQUESTED** [review]
Please fix the empty file problem. Pay a attention to your `git add`.

File: paddlenlp/transformers/squeezebert/modeling.py (hunk 7 above)
Comment (renmada) [reply paddlenlp/transformers/squeezebert/modeling.py:752]: PLZ review latest submit.

**Comment by renmada** [comment]
Please review latest submit.

File: paddlenlp/transformers/squeezebert/modeling.py (hunk 7 above)
Comment (renmada) [reply paddlenlp/transformers/squeezebert/modeling.py:753]: > Need more doc for models for specific tasks

I have not got your message for two days, is there any wrong?

File: paddlenlp/transformers/squeezebert/modeling.py (hunk 2 above)
Comment (smallv0221) [reply paddlenlp/transformers/squeezebert/modeling.py:247]: here

File: examples/language_model/squeezebert/models/squeezebert-mnli/tokenizer_config.json [hunk 8]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (smallv0221) [inline examples/language_model/squeezebert/models/squeezebert-mnli/tokenizer_config.json]: Don't need this file.

File: examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json [hunk 9]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (smallv0221) [inline examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json]: Don't need this file.

**Review by smallv0221 — CHANGES_REQUESTED** [review]
Please delete train_log.txt and dataset file.

File: paddlenlp/transformers/squeezebert/modeling.py (hunk 2 above)
Comment (smallv0221) [reply paddlenlp/transformers/squeezebert/modeling.py:247]: Need to fix this.

File: examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json (hunk 9 above)
Comment (renmada) [reply examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json]: db5a14a6979a52d7b6fa06de8a07ae4
I was told to upload all the relevant files. And he will delete all files after testing.

Comment (smallv0221) [reply examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json]: OK, then wait until testing passes.

**Comment by smallv0221** [comment]
Since your test passed, you can delete all pytorch script, logs and json files

**Comment by renmada** [comment]
> Since your test passed, you can delete all pytorch script, logs and json files

How about remove all files under "examples/language_model/squeeze_bert" ?

**Comment by smallv0221** [comment]
> > Since your test passed, you can delete all pytorch script, logs and json files
> 
> How about remove all files under "examples/language_model/squeeze_bert" ?

Yes. You can remove all examples.

**Comment by renmada** [comment]
> > > Since your test passed, you can delete all pytorch script, logs and json files
> > 
> > 
> > How about remove all files under "examples/language_model/squeeze_bert" ?
> 
> Yes. You can remove all examples.
Done.

**Review by smallv0221 — DISMISSED** [review]
LGTM

**Comment by ZeyuChen** [comment]
@yingyibiao 这个PR有效吗？

**Comment by chenxiaozeng** [comment]
> PR

**Review by smallv0221 — APPROVED** [review]
LGTM

**Comment by smallv0221** [comment]
Please sign our Contributor License Agreement for this pr to be merged
